### DISCUSSION TITLE
SMTP email service "Encryption" field - what goes in here?

### DISCUSSION THREAD

**Discussion by andsat** [mettle/sendportal#280] [Help]
As the title asks, what is supposed to go in the "encryption" text field when setting up a SMTP email service?

I am using mailinabox on a separate (dedicated) server with SMTP and password authentication on port 465.

my current setup requires FORCE_TLS but "plain" login (no OpenSSL verification, no STARTTLS).

I could not find this in the documentation. for clarification, my current setup does not work at all. it doesn't report any errors either. (currently testing sendportal as a potential choice for our org's needs).

**Comment by abrahamgarcia27** [comment]
@andsat did you ever find it?

**Reply by andsat** [reply]
no, I ended up going with listmonk instead.
